src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -1794,6 +1797,8 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 				message: error.localizedDescription,
 				completion: { [weak self] in
 					self?.dismiss()
+					// reset the home badge count
+					self?.badgeWrapper.resetAll()
```

Comment:
You are resetting the total count, that means, if there was a count before that was unrelated to this condition here, will also be reseted.

Reply:
Fixed.